docs/crds/hostrule.md

```diff
@@ -278,6 +279,9 @@ The `loadBalancerIP` field can be used to provide a valid preferred IPv4 address
 
 **Note**: The HostRule CRD is not aware of the misconfigurations while it is being created, therefore the HostRule will be `Accepted` nonetheless.
 
+#### L7Rule 
+
+L7rule field can be used to specify the name of [L7Rule](./l7rule.md) CRD. It is used to modify select VS Properties which are not part of HostRule CRD.
```

Comment:
Lets add comment saying it is applicable only in `v1beta1` hostrule definition

Reply:
added